recipes-extended/xen/blktap3/fix-error-checks.patch

```diff
@@ -0,0 +1,357 @@
+################################################################################
+SHORT DESCRIPTION: 
+################################################################################
+Fix error checks.
+
+################################################################################
+LONG DESCRIPTION: 
+################################################################################
+Replace atoi() with strtol().
+atoi() has limited functionality when it comes to error checks on the input.
+strtol() provides broader error checks on the input.
```

Comment:
While strtol is certainly better than atoi, do we need to worry about this?  Also, there is a lot of duplication, if you create a parse_int function you could centralize all or it.

Reply:
I had introduced this change as part of review fix from @eric-ch while updating OpenXT/blktap3. I can either remove this completely or create parse_int function as you suggested.
@eric-ch: do you have any suggestions ?